### PR TITLE
[Typography] Using safeSharedApplication in test

### PR DESCRIPTION
**Pull request by randallli**
I needed to change this to run test locally in my simulator.

I'm not sure why this wasn't caught in CI.

### CODE REVIEW

File: components/Typography/tests/unit/UIFont+MaterialTypographyTests.m [hunk 1]
```diff
@@ -15,6 +15,7 @@
 #import <XCTest/XCTest.h>
 #import "MDCTypographyMockTraitEnvironment.h"
 #import "MaterialTypography.h"
+#import "MaterialApplication.h"
```
Comment (romoore) [inline components/Typography/tests/unit/UIFont+MaterialTypographyTests.m:17]: Please add this dependency in the Cocoapods and Bazel files.

**Review by romoore — APPROVED** [review]
LGTM, but please update the dependencies.

File: components/Typography/tests/unit/UIFont+MaterialTypographyTests.m (hunk 1 above)
Comment (randallli) [reply components/Typography/tests/unit/UIFont+MaterialTypographyTests.m:17]: Done. added to both.

BTW, It doesn't look like any other components do this for the unit tests in the Cocoapods.

**Comment by randallli** [comment]
`git clang-format $(git merge-base origin/develop HEAD)`
clang-format did not modify any files

https://github.com/material-components/material-components-ios/issues/8814

**Review by bryanoltman — APPROVED** [review]

File: components/Typography/tests/unit/UIFont+MaterialTypographyTests.m (hunk 1 above)
Comment (romoore) [reply components/Typography/tests/unit/UIFont+MaterialTypographyTests.m:17]: That's a shame. It's good to Include What You Use. ;-)

**Review by romoore — APPROVED** [review]

File: MaterialComponents.podspec [hunk 2]
```diff
@@ -44,7 +44,7 @@ Pod::Spec.new do |mdc|
   #     tests.test_spec 'unit' do |unit_tests|
   #       unit_tests.source_files = [
   #         "components/#{component.base_name}/tests/unit/*.{h,m,swift}",
-  #         "components/#{component.base_name}/tests/unit/supplemental/*.{h,m,swift}"
+  #         "components/#{component.base_name}/unit_tests/unit/supplemental/*.{h,m,swift}"
```
Comment (romoore) [inline MaterialComponents.podspec]: Looks like this changed by accident? Please revert if it is.

**Review by jverkoey — DISMISSED** [review]
Adding changes requested bit so this doesn't accidentally land with the requested change not having been made.

File: MaterialComponents.podspec (hunk 2 above)
Comment (randallli) [reply MaterialComponents.podspec]: Done. good catch.

**Comment by randallli** [comment]
> Adding changes requested bit so this doesn't accidentally land with the requested change not having been made.

I saw it and fixed it.

**Pull request merged**